src/components/tree-views-container.tsx

```diff
@@ -333,7 +333,7 @@ const TreeViewsContainer = () => {
             let uploadingElementsInSelectedDirectory = Object.values(uploadingElementsRef.current).filter(
                 (e) => e.directory === selectedDirectoryRef.current?.elementUuid
             );
-            if (uploadingElementsInSelectedDirectory?.length > 0) {
+            if (uploadingElementsInSelectedDirectory) {
```

Comment:
keep the previous test I think to separate both use cases when we have elements in `uploadingElements` and when it's empty